src/components/Anatomy.jsx

```diff
@@ -1,14 +1,33 @@
 import './Anatomy.css';
-import { useRef, useState, useEffect } from 'react';
+import { useRef, useState, useEffect, Suspense } from 'react';
 import { useFrame, Canvas } from '@react-three/fiber'
-import { Grid, OrbitControls, useGLTF, PivotControls, GizmoHelper, GizmoViewport, Center, Text3D, PerspectiveCamera } from "@react-three/drei"
+import { Grid, OrbitControls, useGLTF, PivotControls, GizmoHelper, GizmoViewport, Center, Text3D as Text3DBase, PerspectiveCamera } from "@react-three/drei"
+import {suspend } from 'suspend-react';
+import { TTFLoader } from 'three/examples/jsm/loaders/TTFLoader.js';
 import * as THREE from 'three';
 import axios from 'axios';
 import Confetti from 'react-confetti'
 import useWindowSize from 'react-use/lib/useWindowSize'
 
 const sessionData = JSON.parse(sessionStorage.getItem("sessionData"));
 
+function Text3D({ url, ...props }) {
+  // Suspend while loading and parsing the TTF file.
+  const font = suspend(() => {
+    const loader = new TTFLoader();
+    return new Promise((resolve) => {
+      loader.load(url, resolve)
+    });
+  }, [url]);
+
+  return (
+    // Center component centers the text
+    <Center>
+      {/* Pass the loaded font to the component for rendering */}
+      <Text3DBase font={font} {...props} />
+    </Center>
+  );
+}
```

Comment:
I am not really sure why you are changing the Text3D? 
I think it should work now without any changes required after I added the Inter_Bold.json font file yesterday to main.
Am I missing something here?

Reply:
Okay. I added this approach before merging the file "Inter_Bold.json in `public/` dirctory. I reverted the Anatomy.jsx back to your version. It should work now.